awx.go

```diff
@@ -5,6 +5,9 @@ import (
 	"net/http"
 )
 
+// This variable is to be populated for creating services
```

Comment:
maybe `mandatory fields for services API`?

Reply:
You're right. Fixed it.